auth/server/base/src/main/java/org/wildfly/security/auth/server/MechanismInformation.java

```diff
@@ -53,6 +53,10 @@ public interface MechanismInformation {
      */
     String getProtocol();
 
+    default String getRequestURI() {
```

Comment:
@fjuma We can make use of the existing `getHost` and `getProtocol` methods instead. The URI information would not have a full  requested path that way, just the host and protocol. 

However if we want to avoid adding a new method to the public API we might not have a better choice. So if @OndrejKotek thinks we should not introduce a new API method to the maintenance branch, we can check with the reporter about providing only the host+port information

Reply:
This is not up to me to decide. Open a discussion on https://issues.redhat.com/browse/JBEAP-25337 please as it's up to the QE CP (Peter Mackay, Dan Cihak) and SET team to ack such changes.